en/micro-integrator/docs/use-cases/examples/template_examples/using-endpoint-templates.md

```diff
@@ -28,50 +28,117 @@ For example, let's say we have two default endpoints with following hypothetical
       <retriesBeforeSuspension>3</retriesBeforeSuspension>
       <retryDelay>0</retryDelay>
     </markForSuspension>
-  </default>
+  </address>
 </endpoint>
 ```
 
 We can see that these two endpoints have different set of error codes and different progression factors for suspension. Furthermore, the number of retries is different between them. By defining an endpoint template, these two endpoints can be converged to a generalized form. This is illustrated in the following:
 
 ```
-<template name="ep_template" xmlns="http://ws.apache.org/ns/synapse">
-  <parameter name="codes"/>
-  <parameter name="factor"/>
-  <parameter name="retries"/>
-  <endpoint name="$name">
-    <default>
-        <suspendOnFailure>
-          <errorCodes>$codes</errorCodes>
-          <progressionFactor>$factor</progressionFactor>
-        </suspendOnFailure>
-        <markForSuspension>
-          <retriesBeforeSuspension>$retries</retriesBeforeSuspension>
-          <retryDelay>0</retryDelay>
-        </markForSuspension>
-    </default>
-  </endpoint>
+<template xmlns="http://ws.apache.org/ns/synapse" name="ep_template">
+   <parameter name="name"/>
+   <parameter name="uri"/>
+   <parameter name="codes"/>
+   <parameter name="factor"/>
+   <parameter name="retries"/>
+   <endpoint name="$name">
+      <address uri="$uri">
+         <suspendOnFailure>
+            <errorCodes>$codes</errorCodes>
+            <progressionFactor>$factor</progressionFactor>
+         </suspendOnFailure>
+         <markForSuspension>
+            <retriesBeforeSuspension>$retries</retriesBeforeSuspension>
+            <retryDelay>1</retryDelay>
+         </markForSuspension>
+      </address>
+   </endpoint>
 </template>
 ```
 
 !!! Note
-    - The Endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
+    - The endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
```

Comment:
This paragraph uses a lot of passive voice. If this template is something created by the user and not something pre-configured, I suggest the following:

"The endpoint template uses parameters as inputs. Then you are using the `$' prefix to refer to these parameters within the template.  Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). e.g., You can refer to a parameter named `codes' as `$codes'."

Reply:
Done